README.adoc

```diff
@@ -71,22 +71,20 @@ include::{common-includes}/gitclone.adoc[]
 
 == Starting the service
 
+Navigate to the `start` directory to begin.
+
 This guide is already setup with a general application. As you progress through the guide
-you will make updates to the code directly, and then push updates to the server
-so you can see the results.
+you will make updates to the code directly and automatically see the results using development mode.
```

Comment:
Change
"This guide is already setup with a general application. As you progress through the guide you will make updates to the code directly and automatically see the results using development mode."
To
"This guide is already setup with a general application. As you progress through the guide, you will update the code directly and automatically see the results by using development mode."